pkg/graph/schema.graphql

```diff
@@ -286,6 +286,46 @@ type CedarURL {
   urlHostingEnv: String
 }
 
+"""
+CedarExchangeTypeOfDataItem is one item of the TypeofData slice in a CedarExchange
+"""
+type CedarExchangeTypeOfDataItem {
+  id: String
+  name: String
+}
+
+"""
+CedarExchange represents info about how data is exchanged between a CEDAR system and another system
```

Comment:
This comment should probably document how to check whether a `cedarSystemId` used to query is the sender or receiver, basically the same info that's in this PR's description.

Reply:
I have a few ideas of what this maybe could look like, and would love to get feedback!

- Perhaps we could document it in the GQL schema itself as a comment, that way folks implementing it are aware?
- We could also consider adding the field ourselves until it's exposed by CEDAR. Nothing stops us from not having the GQL be 1:1 with the CEDAR response, so we could always add an `enum ExchangeDirection` and tack it onto the response!

I think I'd actually prefer the latter, as it makes less logic on the frontend, and probably isn't a huge lift on the backend (plus, if CEDAR introduces that field, we can always just replace it!)

Reply:
@ClayBenson94 I actually had an enum like that before, but it just felt like a lot of extra code to do something that can be trivially filtered for in one line on the front end. Are we over thinking this?

Reply:
@jstrang-oddball Yeah - could you perhaps just add some docs to clarify this behavior somewhere?